src/test/java/com/study/racing/StringTest.java

```diff
@@ -0,0 +1,53 @@
+package com.study.racing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("\uD83D\uDE80String 클래스에 대한 학습 테스트 - 1단계")
+class StringTest {
+	String regex = ",";
+
+	@Test
+	@DisplayName("요구사항 1-1")
```

Comment:
사용자가 테스트 코드로 들어왔을때 어떤 기능의 테스트인지
한눈에 보면 좋을 것 같아요!
```suggestion
	@Test
@DisplayName("문자열 자르기")
void split() {
}
```
```suggestion
@Test
void 문자열_자르기() {
}
```